io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/Pdf4QtViewerProfi",
+    "build-options": {
+	"build-args": [ "--share=network" ]
+	},
```

Comment:
```suggestion
```

this is not allowed and will always fail.

Reply:
OK, removed.